### ISSUE TITLE
Better file/folder names perhaps. 

### ISSUE THREAD

**Issue by HappyZombies** [HappyZombies/Sign_Up#1]
I'm really not that creative when it comes to files names. And variable names. If you feel like changing the name of something feel free to.

**Comment by codermason** [comment]
You should really look into proper naming convention in Java. I know it may seem redundant to follow some guidelines on naming classes/variables, but it is a lot easier to read and understand your code when you do (at least for me).

**Comment by HappyZombies** [comment]
Yeah :( it's just a lot of times I just tell myself 'let me call it this for now' and I don't change it. I think I'll go ahead and fix that today when I get the chance.
